plio/management/commands/pliooauth2application.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+import os
+from oauth2_provider.models import Application
+
+
+class Command(BaseCommand):
+    help = "Creates OAuth2 client credentials"
+
+    def handle(self, *args, **options):
+        default_oauth2_client_setup = os.environ.get(
+            "DEFAULT_OAUTH2_CLIENT_SETUP", False
+        )
+        default_oauth2_client_id = os.environ.get("DEFAULT_OAUTH2_CLIENT_ID")
+        default_oauth2_client_secret = os.environ.get("DEFAULT_OAUTH2_CLIENT_SECRET")
```

Comment:
plus, these values won't have to be extracted if `default_oauth2_client_setup` is false, right?

So, we'll have to set these variables while deploying on staging too, right? need to ensure that we don't forget it - so, better to set these variables before merging itself.

Reply:
It is optional but it is present in the `entrypoint.sh` so it will get run anyhow right?
But that shouldn't be an issue for the cases where we already have a client id and secret information in the .env file, in that case, it'll just skip and won't create. So no need of any change in local/staging/prod

This is specifically so that the cypress can set up a client when setting up the repo for first time.
Am i correct?